src/views/inspector/InspectorViewProvider.ts

```diff
@@ -90,18 +91,36 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
       this.matches = await usagesCLIController.usagesKeys()
       this.environments = await environmentsCLIController.getAllEnvironments()
       this.featureConfigsMap = {}
-      
+
       this.selectedKey = ''
       // Send variables data to the webview to initialize fuse
-      this.postMessageToWebview({type: 'variables', value: JSON.stringify(this.orderedVariables)})
-      this.postMessageToWebview({type: 'features', value: JSON.stringify(this.orderedFeatures)})
+      this.postMessageToWebview({
+        type: 'variables',
+        value: JSON.stringify(this.orderedVariables),
+      })
+      this.postMessageToWebview({
+        type: 'features',
+        value: JSON.stringify(this.orderedFeatures),
+      })
     } catch (e) {
       vscode.window.showErrorMessage(
         `Error initializing features and variables in inspector: ${e}`,
       )
     }
   }
 
+  private getLoggedInFolders() {
```

Comment:
new function to get logged in folders

Reply:
I could see this being a useful util in the other views